### DISCUSSION TITLE
What comes next in this sequence?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#967] [Galaxy Trivia] [answered]
```
1  1  2  4  3  9  4  61  5  52  6  63  7  94  8 ?
```

**Comment by ParanoidUser** [comment]
⚠️ Attention! Curious minds (@Deepanshu0703 / @Kalyan-velu) needed for a puzzling question.

**Comment by peterbaccaro** [comment]
The next number in the sequence is **46**.

If you consider the number in pairs the right number is the square of the first number but reversed.

The last pair is 8 ?, so the right number is calculated as 8 x 8 = 64, reversed is 46.

**Comment by CelioAmaral** [comment, marked as answer]
1  1  2  4  3  9  4  61  5  52  6  63  7  94  8 ?

1² = 1, which inverted is 1
2² = 4, which inverted is 4
3² = 9, which inverted is 9
4² = 16, which inverted is 61
5² = 25, which inverted is 52
6² = 36, which inverted is 63
7² = 49, which inverted is 94
8² = 64, which inverted is 46
9² = 81, which inverted is 18

The next number in the sequence is 46.
